gotest.sh

```diff
@@ -173,6 +173,7 @@ if [ "$NEED_COVERAGE" == "true" ] || [ "$RACE" == "false" ]; then
         LOGOUTPUT_FLAG="-logoutput=true"
     fi
 
+    check_test_ports
```

Comment:
Why's this needed at all? Haven't seen travis get tripped on port allocation in months.